core/src/toga/app.py

```diff
@@ -78,16 +77,16 @@ class MainWindow(Window):
     _WINDOW_CLASS = "MainWindow"
 
     def __init__(
-        self,
-        id=None,
-        title=None,
-        position=(100, 100),
-        size=(640, 480),
-        toolbar=None,
-        resizeable=True,
-        minimizable=True,
-        factory=None,  # DEPRECATED !
-        on_close=None,
+            self,
+            id=None,
+            title=None,
+            position=(100, 100),
+            size=(640, 480),
+            toolbar=None,
+            resizeable=True,
+            minimizable=True,
+            factory=None,  # DEPRECATED !
+            on_close=None,
```

Comment:
I'm betting you use PyCharm. I can tell, because PyCharm's default indentation rules are Just Wrong. This has been picked up by the pre-commit check run in CI; if you install pre-commit in your local development environment, you'll get the same warning.